### PR TITLE
Coin config additions

### PR DESCRIPTION
**Pull request by Gumbees**
Added vot.json and zel.json to the coins and pool_configs. Please wait until Pull Request #1 is merged. There are some out of date config files compared to that branch.

### CODE REVIEW

**Review by Machotacoz — DISMISSED** [review]

File: pool_configs/vot.json [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    "_comment_tAddress": "transparent address used to send payments, make this a different address, otherwise payments will not send",
+
+    "walletInterval": 2.5,
+
+    "rewardRecipients": {
+        "": 0.2
+    },
+
+    "paymentProcessing": {
+	"minConf": 10,
```
Comment (egyptianbman) [inline pool_configs/vot.json:21]: Can  you fix the alignment on this line?

File: pool_configs/vot.json [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+            "host": "127.0.0.1",
+            "port": 19332,
+            "user": "testuser",
+            "password": "testpass"
+        }
+    },
+
+    "tlsOptions": {
+        "enabled": false,
+        "serverKey":"",
```
Comment (egyptianbman) [inline pool_configs/vot.json:38]: I also fixed lack of space after the colon on several of these in the other PR.

Comment (Gumbees) [reply pool_configs/vot.json:38]: sounds good. i went through and cleaned up the way it looked anyways and a few other things like the correct port numbers. let me know if vot is correct.

File: pool_configs/vot.json (hunk 1 above)
Comment (Gumbees) [reply pool_configs/vot.json:21]: done.

**Review by Gumbees — COMMENTED** [review]
I need to update the default rpcuser and rpcpass.

**Comment by Gumbees** [comment]
v2 has zel and vot in it. deleting.

**Pull request closed without merging**